pyston/pyston_lite/pyston_lite_autoload.pth

```diff
@@ -0,0 +1 @@
+import os, sys; exec("""sys.path.append("/home/kmod/pyston/pyston/pyston_lite"); ("PYSTON" in os.environ) and __import__("pyston_lite").enable()""")
```

Comment:
hard coded user `kmod` in path. Would be nice if this file would get generated or you find another way to get the path.